### DISCUSSION TITLE
Graceful config reload

### DISCUSSION THREAD

**Discussion by f0o** [akvorado/akvorado#1506] [Q&A] [answered]
Hi,

After modifying the config how do you reload the services without bringing restarting them and causing outages?

Thanks in advance!

**Comment by vincentbernat** [comment, marked as answer]
This is not currently possible. Check https://github.com/akvorado/akvorado/issues/55 (but this is unlikely to be implemented for a long time). There is also work in progress to minimize the tasks done by the inlet.

**Reply by f0o** [reply]
Thanks for the quick answer!

I wonder can I spin up two akvorado's side by side that point to the same kafka/clickhouse instance or will that cause issues? Then I could just restart them one by one without loosing flows.

**Reply by vincentbernat** [reply]
It should work. This seems a good idea. I think they will accept to bind to the same port.

**Reply by vincentbernat** [reply]
When you say two instances, you really only need to spawn two inlets. Two consoles would work, but you can't spawn two orchestrators.

**Reply by f0o** [reply]
if I only spawn 2 inlets and restart one of them, will it automatically apply the new config? or do I need to restart orchestrator as well?

**Reply by vincentbernat** [reply]
You need to restart the orchestrator (by default, this is the one distributing the configuration).

**Reply by f0o** [reply]
~~so I restart Orchestrator and then Inlet? or just Orchestrator?~~

Nevermind, Orchestrator first, then rolling update of Inlet/s
